### PR TITLE
Doc: Mention that `zfs.delegate` is best used in conjunction with `security.nesting`

### PR DESCRIPTION
**Pull request by simondeziel**

### CODE REVIEW

File: lxd/storage/drivers/driver_zfs_volumes.go [hunk 1]
```diff
@@ -1621,7 +1621,9 @@ func (d *zfs) commonVolumeRules() map[string]func(value string) error {
 		"zfs.use_refquota": validate.Optional(validate.IsBool),
 		// lxdmeta:generate(entities=storage-zfs; group=volume-conf; key=zfs.delegate)
 		// This option controls whether to delegate the ZFS dataset and anything underneath it to the
-		// container or containers that use it. This allows using the `zfs` command in the container.
+		// container or containers that use it. When used in conjunction with
+		// {config:option}`instance-security:security.nesting`, this allows
+		// using the `zfs` command in the container.",
```
Comment (minaelee) [inline lxd/storage/drivers/driver_zfs_volumes.go:1626]:
```suggestion
		// using the `zfs` command in the container.
```

Comment (minaelee) [reply lxd/storage/drivers/driver_zfs_volumes.go:1626]: Why the `",` at the end of the line? It renders like this in [the preview](https://canonical-ubuntu-documentation-library--14629.com.readthedocs.build/lxd/en/14629/reference/storage_zfs/#storage-zfs-volume-conf:zfs.delegate):

image

**Review by minaelee — DISMISSED** [review]

File: lxd/storage/drivers/driver_zfs_volumes.go (hunk 1 above)
Comment (simondeziel) [reply lxd/storage/drivers/driver_zfs_volumes.go]: Copy-n-pasto, now fixed, thanks for catching!

**Review by tomponline — APPROVED** [review]
Ta

**Pull request merged**